test/project-viewer.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ca-gip.github.com/v1
```

Comment:
shouldn't this be testdata for the test needing it? I also don't see where it's used/tested.